File: src/main/java/com/ambition/business/user/domain/SysRole.java
package com.ambition.business.user.domain;

import com.baomidou.mybatisplus.annotation.IdType;
import java.util.Date;
import com.baomidou.mybatisplus.annotation.TableId;
import java.io.Serializable;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

/**
 * <p>
 * 后台角色表
 * </p>
 *
 * @author wuys
 * @since 2019-12-11
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
public class SysRole implements Serializable {

    private static final long serialVersionUID = 1L;

    @TableId(value = "id", type = IdType.AUTO)
    private Integer id;

    /**
     * 角色名
     */
    private String roleName;

    private Date createTime;

    private Date updateTime;

    /**
     * 是否有效  1有效  2无效
     */
    private String deleteStatus;

    /**
     * 组织id
     */
    private Long groupId;

    /**
     * 组织名称
     */
    private String groupName;

    /**
     * 创建者id
     */
    private Long createUserId;

    private String createUserName;


}

File: src/main/java/com/ambition/business/user/domain/SysRole.kt
package com.ambition.business.user.domain

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import java.io.Serializable
import java.util.*


/**
 * <pre>
 *
 *
 * @title: SysRole
 * @description:
 * @company: 润投科技
 * @author: wuys
 * @datetime: 2019-12-19 10:40
 * </pre>
 */
class SysRole : Serializable {

    @TableId(value = "id", type = IdType.AUTO)
    val id: Int? = null

    /**
     * 角色名
     */
    val roleName: String? = null

    val createTime: Date? = null

    val updateTime: Date? = null

    /**
     * 是否有效  1有效  2无效
     */
    val deleteStatus: String? = null

    /**
     * 组织id
     */
    val groupId: Long? = null

    /**
     * 组织名称
     */
    val groupName: String? = null

    /**
     * 创建者id
     */
    val createUserId: Long? = null

    val createUserName: String? = null

    companion object {

        const val serialVersionUID = 1L
    }


}
